src/bootstrap.php

```diff
@@ -49,12 +50,14 @@
     'api.uri' => $config['api.uri'] ?? 'https://api.elifesciences.org/',
     'api.timeout' => $config['api.timeout'] ?? 1,
     'debug' => $config['debug'] ?? false,
+    'logger.channel' => 'recommendations-api',
```

Comment:
if we intend to require an explicit `channel` value and to use it to discern logs, maybe it should just be the name of the application? `recommendations`

Reply:
I agree, this is just the existing value.